Sources/MintKit/Mint.swift

```diff
@@ -175,7 +176,14 @@ public class Mint {
                     }
                 }
             } catch {
-                throw MintError.repoNotFound(package.gitPath)
+                let repoVersionsPath = packagesPath + package.repoPath + "build"
+                guard let installedVersions = try? repoVersionsPath.children().map({ $0.lastComponent }),
+                    let fallbackVersion = installedVersions.first else {
+                    throw MintError.repoNotFound(package.gitPath)
+                }
+                package.version = installedVersions
```

Comment:
Can we print to stderror here: "Failed to find latest version online. Falling back to local version \(package.version)"

Reply:
Of course. Great suggestion!